scripts/shared/clusters.sh

```diff
@@ -304,6 +302,16 @@ function warn_inotify() {
     fi
 }
 
+function download_ovnk() {
+    if [[ "${cluster_cni[$cluster]}" == "ovn" ]]; then
```

Comment:
This won't work properly since you're not calling it in a context where `$cluster` is set.

You should either call it from `deploy_kind_ovn` and risk parallel process forks overwriting each other, or check if any of the clusters needs ovn (basically if `${cluster_cni[*]}` contains `ovn`).

Also fail fast is easier to read so if no cluster needs ovn you can `return` from the function, otherwise do what you need.

Reply:
right so calling it in deploy_kind_ovn creates issues, I went ahead and checked if ${cluster_cni[*]} contains "ovn" and made it so that we fail fast if possible